bpf/tests/tc_egressgw.c

```diff
@@ -0,0 +1,187 @@
+// SPDX-License-Identifier: (GPL-2.0-only OR BSD-2-Clause)
+/* Copyright Authors of Cilium */
+
+#define HAVE_LPM_TRIE_MAP_TYPE
+#include "common.h"
+
+#include <bpf/ctx/skb.h>
+#include "pktgen.h"
+#include "config_replacement.h"
+
+/* Set ETH_HLEN to 14 to indicate that the packet has a 14 byte ethernet header */
```

Comment:
nit: s/byte/bytes, s/header/headers

Reply:
Given that "14 byte" is used as an adjective here, the singular form should be correct as well